platform/windows/key_mapping_windows.cpp

```diff
@@ -411,3 +414,7 @@ unsigned int KeyMappingWindows::get_scansym(unsigned int p_code, bool p_extended
 
 	return keycode;
 }
+
+bool KeyMappingWindows::is_extended_key(unsigned int p_code) {
+	return _extended_keys.count(p_code) > 0 ? true : false;
```

Comment:
`_extended_keys` never used anywhere else, IMO this can be written as
```c++
return p_code == VK_INSERT || p_code == VK_DELETE || ...;
```

Reply:
BTW, this line is `cond ? true : false`, the ternary operator is unnecessary, `cond` is already a boolean.